service/lib/agama/software/manager.rb

```diff
@@ -152,6 +154,8 @@ def probe
       def initialize_target
         # create the zypp lock also in the target directory
         ENV["ZYPP_LOCKFILE_ROOT"] = TARGET_DIR
+        # controls if grub-bls should be used. We enable it only for some products.
+        ENV["ENV_NO_BLS_BOOT"] = "no"
```

Comment:
I like the approach of injecting the env variable. But I wonder why this is at software manager instead of the storage one.